### PR TITLE
LPS-170310 Provide way to redirect to “Add entry page“

### PR DESCRIPTION
**Pull request by seltong**
Related ticket: https://issues.liferay.com/browse/LPS-170310

### CODE REVIEW

File: modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/entries/display/context/ObjectEntryDisplayContext.java [hunk 1]
```diff
@@ -329,21 +333,76 @@ public ObjectLayoutTab getObjectLayoutTab() throws PortalException {
...
+						_objectRequestHelper.getCurrentURL()
+					).setParameter(
+						"objectDefinitionId",
+						objectDefinition2.getObjectDefinitionId()
+					).setWindowState(
+						WindowState.MAXIMIZED
+					).buildString());
+				dropdownItem.setLabel(
+					LanguageUtil.get(
+						_objectRequestHelper.getRequest(), "create-new"));
```
Comment (guilhermedcamacho) [inline modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/entries/display/context/ObjectEntryDisplayContext.java:395]: Hey @seltong , I think you should validate if the user has the permission do add entry before adding the option to menu.
We have this acceptance criteria in the story: "The create new option should only appear if the user has permission to add entries in the related object"

**Review by guilhermedcamacho — CHANGES_REQUESTED** [review]

**Comment by seltong** [comment]
ci:test:sf

**Comment by seltong** [comment]
ci:stop

**Comment by seltong** [comment]
ci:test:sf

**Comment by guilhermedcamacho** [comment]
See #1505

**Pull request closed without merging**